config/locales/hyrax.zh.yml

```diff
@@ -827,10 +823,10 @@ zh:
         identifier: 一个独特的句柄用于标识收藏集。 例如用于标识期刊文章的DOI，或者标识一本书的ISBN或者OCLC系统编号。
         keyword: 用于描述和检索收藏集内容的的词或词组。
         language: 收藏集内容使用的语言。
-        license: 用于管理收藏集访问权限的许可与发布信息。 可从下拉列表中选择。
         publisher: 出版收藏集的个人或团体。 一般来说是机构单位。
         related_url: 相关链接，用于链接与收藏集相关的网页或其它具体内容(音频、视频、PDF文件)。例如一个研究项目的网址，收藏集是此研究项目的成果
         resource_type: 预定类型用于描述上传内容类型，例如&quot;文章&quot; or &quot;数据集。&quot; 可以选择多项类型。
+        license: 用于管理收藏集访问权限的许可与发布信息。 可从下拉清单中选择。
```

Comment:
Please move `license` back to its location between `language` and `publisher`.

Reply:
In English version, `license` is below `resource_type`: https://github.com/samvera/hyrax/blob/master/config/locales/hyrax.en.yml#L839